test/behaviour/protractor.conf.js

```diff
@@ -9,5 +9,9 @@ exports.config = {
 		slow: 3000
 	},
 	specs: ['**/*.js'],
-	exclude: ['**/*.conf.js']
+	exclude: ['**/*.conf.js'],
+	mocks: {
+		dir: 'mocks',
+		default: ['default']
+	}
```

Comment:
This was already removed in 2cb90f1b81b0 (but is currently not merged to this branch).